lib/server.coffee

```diff
@@ -119,35 +147,41 @@ WebApp.connectHandlers.use (req, res, next) ->
 			# More info: https://groups.google.com/forum/#!topic/meteor-core/uZhT3AHwpsI
 			if phantomJsArgs.indexOf('--ssl-protocol=') == -1
 				phantomJsArgs += ' --ssl-protocol=TLSv1'
+			# Support all kind of SSLs
+			if phantomJsArgs.indexOf('--ignore-ssl-errors=') == -1
+				phantomJsArgs += ' --ignore-ssl-errors=true'
+			# to allow redirects on some systems we need set --web-security to false
+			if Spiderable.allowRedirects and phantomJsArgs.indexOf('--web-security=false') == -1
+				phantomJsArgs += ' --web-security=false'
 			# Run phantomjs.
-			child_process.exec 'phantomjs ' + phantomJsArgs + ' ' + PHANTOM_SCRIPT + ' ' + JSON.stringify(url), {
+			child_process.exec "phantomjs #{phantomJsArgs} #{PHANTOM_SCRIPT} #{JSON.stringify(url)}"
+			,
 				timeout: REQUEST_TIMEOUT_IN_MILLISECONDS
 				maxBuffer: MAX_BUFFER
-			}, (error, stdout, stderr) ->
-				bindEnvironment( ->
-					if !error and /<html/i.test(stdout)
-						res.writeHead 200, 'Content-Type': 'text/html; charset=UTF-8'
-						if Spiderable.debug
-							console.info 'Spiderable successfully completed for url: ', url
-						cacheCollection.upsert { hash: hash },
-							'$set':
-								hash: hash
-								url: url
-								content: stdout
-								createdAt: new Date
-						res.end stdout
-					else
-						# If phantomjs is failed. Don't send the error, instead send the normal page.
-						if Spiderable.debug
-							console.warn 'Spiderable failed for url: ', url
-						if error and error.code == 127
-							console.warn 'spiderable: phantomjs not installed. Download and install from http://phantomjs.org/'
+			, 
+				(error, stdout, stderr) ->
+					bindEnvironment ->
+						if !error
+							output = JSON.parse stdout.replace /^(?!(\{.*\})$)(.*)|\r\n/gim, ''
```

Comment:
What are you trying to remove in this regex?
I will add a comment here, but I am unsure why we need the replace part.